labscript_devices/IMAQdxCamera/blacs_workers.py

```diff
@@ -139,6 +139,7 @@ def __init__(self, serial_number):
         )
         # Keep an img attribute so we don't have to create it every time
         self.img = nv.imaqCreateImage(nv.IMAQ_IMAGE_U16)
+        self.exception_on_failed_shot = True
```

Comment:
Does the mock camera also need this definition?

Reply:
You are correct.  I didn't realize that MockCamera was being used as a self.camera attribute in the main camera class.  I just pushed this change.